definitions/json/EventToComplexTypes/EventToComplexTypes-WA.json

```
@@ -704,27 +704,37 @@
     "CaseEventID": "draftAndSignJudgement",
     "CaseFieldID": "draftAndSignJudgement",
     "ListElementCode": "isJudgement",
-    "EventElementLabel": "Is this a Judgement?",
+    "EventElementLabel": "Is this a Judgment?",
     "FieldDisplayOrder": 1,
     "DisplayContext": "MANDATORY",
     "Publish": "Y"
   },
+
   {
     "ID": "DraftAndSignJudgement",
     "CaseEventID": "draftAndSignJudgement",
-    "CaseFieldID": "draftAndSignJudgement",
-    "ListElementCode": "draftAndSignJudgementDocument",
-    "EventElementLabel": "Document upload",
+    "CaseFieldID": "DraftAndSignJudgementDocuments",
+    "ListElementCode": "uploadedDocument",
```

You are referencing a field which is not part of the complex type which is why it doesn't show in table. You have been saving the document to a field outside of the complex type
```suggestion
    "CaseFieldID": "draftAndSignJudgment",
    "ListElementCode": "draftAndSignJudgementDocuments",
```